File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/api/message.py
from typing import Any, Dict, List, Optional
from ..http_client import HttpClient
from ..models import CountClaudeTokensPostRequest, CountClaudeTokensPostResponse, CreateClaudeMessageRequest, CreateClaudeMessageResponse

class MessageApi:
    """message API client."""
    
    def __init__(self, client: HttpClient):
        self._client = client

    def count_claude_tokens(self, body: CountClaudeTokensPostRequest) -> CountClaudeTokensPostResponse:
        """Count Claude tokens"""
        return self._client.post(f"/ai/v3/messages/count_tokens", json=body)

    def create_claude(self, body: CreateClaudeMessageRequest, headers: Optional[Dict[str, str]] = None) -> CreateClaudeMessageResponse:
        """Create Claude message"""
        return self._client.post(f"/ai/v3/messages", json=body, headers=headers)

File: sdkwork-ai-sdk-flutter/lib/src/api/message.dart
import '../http/client.dart';
import '../models.dart';

class MessageApi {
  final HttpClient _client;
  
  MessageApi(this._client);

  /// Count Claude tokens
  Future<CountClaudeTokensPostResponse?> countClaudeTokens(CountClaudeTokensPostRequest body) async {
    final response = await _client.post(ApiPaths.aiPath('/messages/count_tokens'), body: body, contentType: 'application/json');
    return response is CountClaudeTokensPostResponse ? response : null;
  }

  /// Create Claude message
  Future<CreateClaudeMessageResponse?> createClaude(CreateClaudeMessageRequest body, Map<String, String>? headers) async {
    final response = await _client.post(ApiPaths.aiPath('/messages'), body: body, headers: headers, contentType: 'application/json');
    return response is CreateClaudeMessageResponse ? response : null;
  }
}
